pkg/event/event.go

```diff
@@ -23,11 +23,16 @@ import (
 	"github.com/redhat-cne/sdk-go/pkg/types"
 )
 
+const (
+	// EVENT is the generic event type
+	EVENT = "com.redhat.cne.event"
+)
+
 // Event represents the canonical representation of a Cloud Native Event.
 // Event Json  payload is as follows,
 //{
 //	"id": "5ce55d17-9234-4fee-a589-d0f10cb32b8e",
-//	"type": "event.synchronization-state-chang",
+//	"type": "com.redhat.cne.ptp_event",
```

Comment:
leave it as event.synchronization-state-change , don't use `redhat`

Reply:
done